### ISSUE TITLE
Player is locked and crashes on join to server when picking up large inventory

### ISSUE THREAD

**Issue by BrailleCortex** [Tschipp/CarryOn#386]
### Expected Behavior
Can pick up and place down a large inventory block

### Actual Behavior
When attempting to pick up a Quark Storage Crate (Quark block that stores 1,000 items regardless of stack size) filled with ~200 full beehives on a server, I get the following error:
![image](https://user-images.githubusercontent.com/39738209/165174654-f88878c1-0905-4ea8-a2f1-5cef2636f6c0.png)

### Steps to Reproduce

I have not attempted to reproduce this issue, given 1) the large amount of time it would take to create over 200 occupied beehives again, and 2) the wish to not brick another player. However, given the issue, I assume the steps to reproduce would be as follows:
1. Load versions of mods listed below, possibly needing a server, have not tested
2. Place large amounts of beehives or other large-NBT items in a crate
3. Pick the crate up with Carry On

### Version of Minecraft, Carry On, Forge

Minecraft 1.16.5
carryon-1.16.5-1.15.5.15
Quark-r2.4-319
Forge 36.2.13

### Screenshots encouraged
![image](https://user-images.githubusercontent.com/39738209/165174654-f88878c1-0905-4ea8-a2f1-5cef2636f6c0.png)
![image](https://user-images.githubusercontent.com/39738209/165180343-de27e8d0-a06c-48f5-abca-f4ebf1e1bf66.png)

**Comment by BrailleCortex** [comment]
A fix to this would be possibly checking if a block about to be carried is too large and not letting it be picked up?

**Comment by BrailleCortex** [comment]
Further note: attempting to join with the bugged player kicks everyone from the server with that message

**Comment by Tschipp** [comment]
😳 Yeah I guess checking the size would maybe be possible, but not sure how the size allocation is calculated.

**Comment by BrailleCortex** [comment]
It's been a bit since I thought about this, I should have commented with my solution when I found it, sorry. What I ended up doing was installing this mod clientside:
https://www.curseforge.com/minecraft/mc-mods/randompatches-forge
that increased the NBT packet size. This worked to let me log in and place the crate down without needing any serverside changes. I did not continue testing to see if this worked reliably, but it seems like it would.
